README.md

```diff
@@ -1,18 +1,24 @@
 # Laserfiche Repository API Client
 
-Use the Laserfiche Repository API to access data in a Laserfiche repository. Import or export files, modify the repository folder structure, read and modify templates and field values, and more.
+Use the Laserfiche Repository API to access data in a Laserfiche repository. Import or export files, modify the
+repository folder structure, read and modify templates and field values, and more.
```

Comment:
Do we want to break this paragraph into two?

Reply:
No. this was done by mistake. I have replaced the original file.